src/components/subtitleeditor/subtitleeditor.scss

```diff
@@ -1,3 +1,11 @@
 .originalSubtitleFileLabel {
     margin-right: 1em;
 }
+
+.subtitleFeaturePillow {
+    background: #38c;
+    color: #fff;
```

Comment:
Let's update these colors to match our brand color:
```suggestion
    background: #00a4dc;
    color: #000;
```

Reply:
Are you sure about the color? it looks a bit "off"? 
![image](https://github.com/jellyfin/jellyfin-web/assets/55142207/105cdf5f-e226-4ce9-98e7-e93180746b79)
compared to previous:
![image](https://github.com/jellyfin/jellyfin-web/assets/55142207/00afc98c-9f58-4dce-b382-49c5ad93bf2f)

Reply:
[Pretty sure](https://jellyfin.org/docs/general/contributing/branding#theme) 🙃